### PR TITLE
Issue #54 LLRender::init crash

### PR DESCRIPTION
**Pull request by akleshchev**
Picking some issue handling logic from maint-x

### CODE REVIEW

**Review by marchcat — APPROVED** [review]

File: indra/llrender/llrender.cpp [hunk 1]
```diff
@@ -879,6 +879,11 @@ void LLRender::init(bool needs_vertex_buffer)
     // necessary for reflection maps
     glEnable(GL_TEXTURE_CUBE_MAP_SEAMLESS);
 
+    if (glGenVertexArrays == nullptr)
```
Comment (marchcat) [inline indra/llrender/llrender.cpp:882]: `&glGenVertexArrays`
otherwise xcode refuses to build this.

Comment (akleshchev) [reply indra/llrender/llrender.cpp:882]: Sorry, missed a commit. it's supposed to be 'windows only'

Comment (akleshchev) [reply indra/llrender/llrender.cpp:882]: Or picket it from a wrong branch...

**Review by marchcat — APPROVED** [review]

**Review by brad-linden — APPROVED** [review]
looks good

**Pull request merged**